spec/factories/workflow_contents.rb

```diff
@@ -3,16 +3,59 @@
     workflow
     language "en"
     strings({
-             "interest.question" => "Draw a circle",
-             "interest.help" => "Duh?",
-             "interest.tools.0.label" => "Red",
-             "interest.tools.1.label" => "Green",
-             "interest.tools.2.label" => "Blue",
-             "shape.question" => "What shape is this galaxy",
-             "shape.help" => "Duh?",
-             "shape.answers.0.label" => "Smooth",
-             "shape.answers.1.label" => "Features",
-             "shape.answers.2.label" => "Star or artifact",
+            "interest.question" => "Draw a circle",
+            "interest.help" => "Duh?",
+            "interest.tools.0.label" => "Red",
+            "interest.tools.1.label" => "Green",
+            "interest.tools.2.label" => "Blue",
+            "shape.question" => "What shape is this galaxy",
+            "shape.help" => "Duh?",
+            "shape.answers.0.label" => "Smooth",
+            "shape.answers.1.label" => "Features",
+            "shape.answers.2.label" => "Star or artifact",
             })
+
+    factory :combo_workflow_content do
```

Comment:
I've been starting to move from nested factories to [traits](https://robots.thoughtbot.com/remove-duplication-with-factorygirls-traits). Probably a better fit here too?